ffi/wgpu.h

```diff
@@ -1558,9 +1558,9 @@ typedef uint32_t WGPUTextureUsage;
  */
 #define WGPUTextureUsage_STORAGE (uint32_t)8
 /**
- * Allows a texture to be a output attachment of a renderpass.
+ * Allows a texture to be an output attachment of a renderpass.
  */
-#define WGPUTextureUsage_OUTPUT_ATTACHMENT (uint32_t)16
+#define WGPUTextureUsage_RENDER_ATTACHMENT (uint32_t)16
```

Comment:
Did you change the file by hand?
I believe the `Cargo.lock` changes are missing from the PR

Reply:
No, I used `make`, I was just intimidated by the huge diff in `Cargo.lock` at first, but now I see that it's mainly dependencies which were updated since the last `Cargo.lock` update. 
Double check please in case I messed up something!